bigbluebutton-html5/imports/ui/components/whiteboard/annotations/poll/component.jsx

```diff
@@ -217,9 +217,10 @@ class PollDrawComponent extends Component {
 
       // first check if we can still increase the font-size
       if (fontSizeDirection === 1) {
-        if (keySizes.width < maxLineWidth && keySizes.height < maxLineHeight
+        if ((keySizes.width < maxLineWidth && keySizes.height < maxLineHeight
           && voteSizes.width < maxLineWidth && voteSizes.height < maxLineHeight
-          && percSizes.width < maxLineWidth && percSizes.height < maxLineHeight) {
+          && percSizes.width < maxLineWidth && percSizes.height < maxLineHeight)
+          && calcFontSize < 100) {
```

Comment:
Do we handle the case where calcFontSize is >= 100?

Reply:
This implementation doesn't allow `calcFontSize` to be > 100